pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerDe.scala

```diff
@@ -282,3 +282,9 @@ object SerDe {
     value.foreach(v => writeBytes(out, v))
   }
 }
+
+object SerializationFormats {
```

Comment:
@concretevitamin Not sure I follow (sealed traits are new to me). Would this be in place of the entire object? 
And how would I then go about referencing this in `RRDD.scala` to get the different string values?